test/functional/page_objects/dashboard_page.js

```diff
@@ -246,9 +246,7 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
     }
 
     async filterVizNames(vizName) {
```

Comment:
How is this function different from the `filterSearchNames` function also provided in dashboard_page?